node/silkworm/rpc/server/backend_calls.cpp

```diff
@@ -36,28 +47,19 @@ void EtherbaseCall::fill_predefined_reply(const EthereumBackEnd& backend) {
     }
 }
 
-EtherbaseCall::EtherbaseCall(boost::asio::io_context& scheduler, remote::ETHBACKEND::AsyncService* service, grpc::ServerCompletionQueue* queue, Handlers handlers)
-    : UnaryRpc<remote::ETHBACKEND::AsyncService, remote::EtherbaseRequest, remote::EtherbaseReply>(scheduler, service, queue, handlers) {
-}
-
-void EtherbaseCall::process(const remote::EtherbaseRequest* request) {
-    SILK_TRACE << "EtherbaseCall::process START request: " << request;
-
+awaitable<void> EtherbaseCall::operator()(grpc::ServerContext& /*server_context*/, remote::EtherbaseRequest& /*request*/,
```

Comment:
There is an unfortunate clash between `grpc::ServerContext `and `silkworm::rpc::ServerContext`. 
Maybe a different name for the latter will help the reader.

Reply:
Yep, you're definitely right. To be completely honest, it's even worse:
- `grpc::ServerContext` from `gRPC` library
- `agrpc::GrpcContext` from `asio-grpc` library
- `boost::asio::io_context` from `Boost Asio` library
- `silkworm::rpc::ServerContext` from our own side

On the other hand, the name used in such libraries demonstrates somehow a well-established way to identify execution loops.

I will try to use a better name for our class in a future PR. Tentative proposal: `ExecutionUnit` or `ExecutionLoop` or just `Context` (yes, I understand this seems not to solve the problem at all)